### PR TITLE
NAS-102114 doc changing the official plugin collection

### PR DESCRIPTION
**Pull request by a-stjohn**

### CODE REVIEW

File: userguide/intro.rst [hunk 1]
```diff
@@ -145,6 +145,10 @@ These major features are new in this version:
   :menuselection:`Plugins`
   page has been redesigned and improved.
 
+* Community created plugins can be easily accessed by changing the
```
Comment (wblock) [inline userguide/intro.rst]:
```suggestion
* Community-created plugins can be accessed by changing the
```

File: userguide/plugins.rst [hunk 2]
```diff
@@ -623,9 +623,10 @@ Official Plugins
 ----------------
 
 :numref:`table %s <plugins-official-plugins>` lists and describes all
-plugins supported by iXsystems. Adding "unofficial" plugins to %brand%
-is supported by following the process outlined in
-:ref:`Create a Plugin <Creating Plugins>`.
+plugins supported by iXsystems. Access the official plugins by choosing
+*iXsystems* from the :guilabel:`Browse a Collection` drop-down. Adding
+"unofficial" plugins to %brand% is supported by following the process
```
Comment (wblock) [inline userguide/plugins.rst]: This sentence is long, complex, and unclear.  Why is unofficial quoted?  "supported by following the process outlined" is complicated, and can be cleared up by rearranging and removing unneeded words: "Unofficial plugins can be added by..." Also, which process in that section?

File: userguide/plugins.rst [hunk 3]
```diff
@@ -786,3 +787,16 @@ platform:
...
   successfully installed at one or more locations.
+
+
+.. index:: community plugins, plugins
+.. _Community Plugins:
+
+Community Plugins
+-----------------
+
+%brand% offers the ability to switch plugin repositories to
```
Comment (wblock) [inline userguide/plugins.rst]: Wordy.
```suggestion
Switching plugin repositories provides access to community-created plugins.
```

File: userguide/plugins.rst [hunk 4]
```diff
@@ -786,3 +787,16 @@ platform:
...
+gain access to community created plugins. Switch the collection of
+plugins available by choosing a repository from the
```
Comment (wblock) [inline userguide/plugins.rst]: What are the repos available in the dropdown?  Should we tell them to choose "community-created"?  "Switch" is not clear here.  We should just tell them that there are different repos available and what they provide.

**Comment by dlavigne** [comment]
This whole section was rewritten for BETA1 with https://github.com/freenas/freenas-docs/commit/f9f1fbd584a1e0a37197cf86415e83f27ef3edfc#diff-332aec4162f72fada9c3aa0287d1e95a and https://github.com/freenas/freenas-docs/commit/f517fef86e9345c4c2f29c18fc634acade7f3514#diff-332aec4162f72fada9c3aa0287d1e95a

**Pull request closed without merging**